x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/query_bar/query_bar.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { DataViewBase } from '@kbn/es-query';
+import React, { useMemo, useState } from 'react';
+import { SearchBar, TimeHistory } from '../../../../../../../../src/plugins/data/public';
+import { Storage } from '../../../../../../../../src/plugins/kibana_utils/public';
+
+type QueryLanguageType = 'lucene' | 'kuery';
+
+export function QueryBar({
+  dataViews,
+  rangeFrom,
+  rangeTo,
+  onQueryChange,
+  query,
+}: {
+  dataViews: DataViewBase[];
+  rangeFrom?: string;
+  rangeTo?: string;
+  query?: string;
+  onQueryChange: ({}: {
+    dateRange: { from: string; to: string; mode?: 'absolute' | 'relative' };
+    query?: string;
+  }) => void;
+}) {
+  const timeHistory = useMemo(() => {
+    return new TimeHistory(new Storage(localStorage));
```

Comment:
Will this use the same localstorage space for every instance of QueryBar? What happens if we use the same component in different places in the app?
I didn't work with Timehistory before so maybe my concern is futile.

Reply:
Honestly, this isn't used anymore. The initial direction changed a few times during the development of this PR and this logic will be added later. I'll remove these files